test/invoker_test.rb

```diff
@@ -682,6 +684,11 @@ def readlines
       else
         ENV.delete 'TZ'
       end
+      if old_source_date_epoch
+          ENV['SOURCE_DATE_EPOCH'] = old_source_date_epoch
+      else
+          ENV.delete 'SOURCE_DATE_EPOCH'
```

Comment:
I don't think it's necessary to delete the SOURCE_DATE_EPOCH environment variable after the test since Asciidoctor doesn't set this value. Can you remove it and retest?